jumpscale/packages/backup/services/system_backup.py

```diff
@@ -0,0 +1,107 @@
+"""
+----------------------------------------------------------------------
+# THE SYSTEM BACKUP SERVICE
+----------------------------------------------------------------------
+this service will run in the background and execute a backup job for the system paths every hour
+
+Examples:
+## run the backup service jobs once manually
+JS-NG> j.packages.backup.SystemBackupService().job()
+
+## adding the backup service manually to the servicemanager, although this would done automatically when threebot start
+```python
+JS-NG> service_manager = j.tools.servicemanager.new('system_backup_service')
+JS-NG> service_manager.add_service('system_backup_service', j.sals.fs.expanduser('~/projects/js-sdk//jumpscale/packages/backup/services/system_backup.py'))
+```
+## for how to create a backup jobs, check backupjob sal docs
+"""
+
+from jumpscale.loader import j
+from jumpscale.tools.servicemanager.servicemanager import BackgroundService
+from jumpscale.tools.notificationsqueue.queue import LEVEL
+
+
+class SystemBackupService(BackgroundService):
+    # we will use this  pre defined BackupJop if exists, else we will define it using the info from next section
+    BACKUP_JOB_NAME = "systembackupjob"
+
+    # system BackupJob info
+    ## this ResticRepo instance must be preconfigured and exist.
+    RESTIC_CLIENT_NAMES = ["systembackupclient"]
+    ## paths to include in the BackupJob
+    BACKUP_JOP_PATHS = ["~/.config/jumpscale/", "~/sandbox/cfg/"]
```

Comment:
typo, `job`, maybe we can include `sshkeys` too, as user can add keys to admin dashboard

Reply:
What do you think about this suggestion @xmonader?
regarding the threebot admin who will configure his restic backend should we backup the `.ssh/` dir or just the `authorized_keys`?

Reply:
if exists I believe yes

Reply:
The `ssh` directory added, Thanks @waleedhammam @xmonader :+1: